test/nodes/test_web_retriever.py

```diff
@@ -1,163 +1,246 @@
 import os
-from typing import Dict, Tuple
-
-
+from unittest.mock import MagicMock, patch, Mock
+from test.conftest import MockDocumentStore
 import pytest
-import requests
-from boilerpy3.extractors import ArticleExtractor
 
 from haystack import Document, Pipeline
-from haystack.nodes import WebSearch, WebRetriever, PromptNode
+from haystack.document_stores.base import BaseDocumentStore
+from haystack.nodes import WebRetriever, PromptNode
+from haystack.nodes.preprocessor import PreProcessor
+from haystack.nodes.retriever.web import SearchResult
+from test.nodes.conftest import example_serperdev_response
```

Comment:
If this constant is only used in this test, I would prefer to have it here in this file.

Reply:
The plan is to use it in serper dev unit tests which are currently integration only.